samples/ck-rx65n/basic-sample/src/iotconnect_app.c

```diff
@@ -157,13 +158,41 @@ static void publish_telemetry() {
 }
 
 /* Include the sample.  */
-bool app_startup(NX_IP *ip_ptr, NX_PACKET_POOL *pool_ptr, NX_DNS *dns_ptr) {
+static bool app_startup(NX_IP *ip_ptr, NX_PACKET_POOL *pool_ptr, NX_DNS *dns_ptr) {
     printf("Starting App Version %s\r\n", APP_VERSION);
     IotConnectClientConfig *config = iotconnect_sdk_init_and_get_config();
     azrtos_config.ip_ptr = ip_ptr;
 	azrtos_config.pool_ptr = pool_ptr;
 	azrtos_config.dns_ptr = dns_ptr;
 
+#ifdef IOTCONNECT_INTERACTIVE_SYMMETRIC_KEY
+    static char iotconnect_cpid[64] = {0};
+    static char iotconnect_env[64] = {0};
+    static char iotconnect_duid[64] = {0};
+    static char iotconnect_symmetric_key[256] = {0};
+
+    printf("Type the IOTCONNECT_CPID:\r\n");
```

Comment:
So ther user is supposed to do this every time the board boots? Isn't there a way to store this in flash?

Reply:
Yes, it's just for an initial quickstart. Also helps if/when you mis-type any information.

Reply:
Ok, I did not define requirements. I assume Kranti is ok with this.